### ISSUE TITLE
How to print step loss during train?

### ISSUE THREAD

**Issue by FuYanzhe2** [google-research/bert#265]
I want to print training loss at each step and I have used tf.train.LoggingTensorHook like this:
![image](https://user-images.githubusercontent.com/45530174/49939485-f5532e00-ff17-11e8-864c-eca0cb02d051.png)
but I haven't get any info!!!!

**Comment by HaishuoFang** [comment]
I use tf.estimator.EstimatorSpec with above code.It works

**Comment by xwzhong** [comment]
https://github.com/google-research/bert/issues/70 check this

**Comment by qiunian711** [comment]
This method is very useful to me, thank you very much.

**Comment by BucherLi** [comment]
what is the mean of every_n_iter and how to set it?

**Comment by guotong1988** [comment]
Mark
